### DISCUSSION TITLE
Does Notarizing after EDDSA signing affect signature validation in appcast.xml?

### DISCUSSION THREAD

**Discussion by nishaanth-ms-21155** [sparkle-project/Sparkle#2664] [Q&A] [answered]
I am compiling an app in another machine, created a dmg, codesigned and eddsa signed, I bring it into my machine to test, the appcast finds the signature invalid. I sign it again using eddsa but am getting a new signature. The only thing pretty much happens is that the app is notarized and stapled. What is going wrong here?

**Comment by zorgiepoo** [comment]
Code signing a dmg modifies the disk image. Need to EdDSA sign afterwards. Not sure if stapling modifies it too (probably).

**Reply by nishaanth-ms-21155** [reply, marked as answer]
It doesn't. It does change the content.
